docs/.vitepress/theme/styles/vars.css

```diff
@@ -70,12 +70,6 @@
   --vp-custom-block-tip-bg: var(--vp-c-brand-dimm);
 }
 
-.dark {
-  --vp-custom-block-tip-border: var(--vp-c-brand);
-  --vp-custom-block-tip-text: var(--vp-c-brand-lightest);
-  --vp-custom-block-tip-bg: var(--vp-c-brand-dimm);
-}
```

Comment:
I think we still need this part. The text in an info block is hard to read.
![image](https://user-images.githubusercontent.com/49056869/213115093-b7c8ceef-7fd9-44c7-8d59-43663e72cf5a.png)
https://deploy-preview-11720--vite-docs-main.netlify.app/plugins/

Maybe the `--vp-custom-block-tip-text` part is the only needed part.
https://github.com/vitejs/vite/pull/9055/files#diff-9565ca77da243a5172a9f9016b05a69ed7c8726a86d7b04df12807bd2d26dce6R75

Reply:
Oops, sorry about that and nice catch 👀 I'll revert this 👍

Reply:
Done! You're right that we only need `--vp-custom-block-tip-text` but I brought back all 3 because it looks better in terms of consistency with the light mode definitions 👀